suzieq/poller/controller/source/native.py

```diff
@@ -30,6 +31,9 @@ def _validate_config(self, input_data: dict):
         if not isinstance(input_data.get('hosts'), list):
```

Comment:
You are here checking if the host list is actually a list, but if the `hosts` is missing, it always return: `Hosts must be a list`. Now I think it is better moving the check at line 34 before, so that if hosts is not defined or it is an empty list it returns `False`, and we might return something like: `The field hosts with the list of nodes to poll must be provided`, then we can check if the `hosts` field is actually a list

Reply:
You right. I fix it right now